wemake_python_styleguide/visitors/ast/keywords.py

```diff
@@ -301,6 +302,21 @@ def _check_for_violations(self, names, return_sub_nodes, returns) -> None:
                     ),
                 )
 
+    def _check_return_at_the_end(self, node):
```

Comment:
This function should be typed.
Like so: `_check_return_at_the_end(self, node: AnyFunctionDef) -> None:`

Reply:
@sobolevn added the change.